src/config/app-config-endpoints.html

```diff
@@ -6,6 +6,39 @@
     domain: '.domain.com'
   };
 
+  function polyfillLocationOrigin() {
+    if (window.location.origin) { return; }
+
+    window.location.origin = window.location.protocol + "//" + window.location.hostname + (window.location.port ? ":" + window.location.port : "");
+  }
+
+  function resolveApiBaseUrl() {
```

Comment:
No reason to parse the origin when the browser does it for us, more room for error.
Should only use config IF on localhost.